Create Issue tracker template (#201)
camaraproject/APIBacklog: .github/ISSUE_TEMPLATE/issue_tracker_template.md

> <!-- API & repository Tracker Issue -->

Reviewer: Add the needed meta-data for the template

```suggestion
---
name: Onboarding Tracker⛴️
about: To be created by API Backlog codeowners after a new repository got approved
title: '[Onboarding Tracker] $repo name$'
labels: 'API Onboarding Tracker'
assignees: ''

---

<!-- API & repository Tracker Issue -->
```

**API information**
- **Name**: XXXX
- **Mail List:** XXXX
- **Repository**: XXXX
- **Wiki Page**: XXXX
- **API owner**: TBD
- **API proposal**: https://github.com/camaraproject/APIBacklog/blob/main/documentation/XXXX.md
- **Original Issue**: https://github.com/camaraproject/APIBacklog/issues/XXXX

**Action List**

> - [ ] Get nominations of initial code owner(s) (one mandatory) (API owner)
> - [ ] Create repository
> - [ ] Update of APIBacklog.md with new status
> - [ ] Update CODEOWNERS file with PR
> - [ ] Optional: get nominations of initial Maintainers (API owner)
> - [ ] Update Maintainers.md file with PR
> - [ ] Create wiki page
> - [ ] Create mailing list
> - [ ] Create README with links to wiki page and mailing list
> - [ ] Update README with API information (API owner)
> - [ ] Update of CAMARA presentation and website

Reviewer: Due to the automation of repository creation it is preferred to have the wiki page and mailing list, I propose the following order, saving the PRs to update the README. Also moving the maintainers.md topics to the end, I even see that they can be omitted completely as not necessary within the onboarding. 

```suggestion
- [ ] Get nominations of initial code owner(s) (one mandatory) (API owner)
- [ ] Update of APIBacklog.md with new status ("Onboarding")
- [ ] Create wiki page and mailing list (send mail to support@camaraproject.org)
- [ ] Create repository (by Admin, using automation)
- [ ] Update README with API information (API owner)
- [ ] Update of CAMARA presentation and website (create issue within Marketing repository)
- [ ] Update of APIBacklog.md with new status ("Onboarded" or similar)
- [ ] Optional: get nominations of initial Maintainers (API owner)
- [ ] Update Maintainers.md file with PR
```

---
name: Onboarding Tracker⛴️
about: To be created by API Backlog codeowners after a new repository got approved
title: '[Onboarding Tracker] $repo name$'
labels: 'API Onboarding Tracker'
assignees: ''

---

<!-- API & repository Tracker Issue -->
**API information**
- **Name**: XXXX
- **Mail List:** XXXX
- **Repository**: XXXX
- **Wiki Page**: XXXX
- **API owner**: TBD
- **API proposal**: https://github.com/camaraproject/APIBacklog/blob/main/documentation/XXXX.md
- **Original Issue**: https://github.com/camaraproject/APIBacklog/issues/XXXX

**Action List**
- [ ] Get nominations of initial code owner(s) (one mandatory) (API owner)
- [ ] Update of APIBacklog.md with new status ("Onboarding")

> - [ ] Create wiki page and mailing list (send mail to support@camaraproject.org)

Reviewer: @wrathwolf @eharrison24 

Should we:
a) split this line into two different action for wiki page and mailing list
b) is it enough that you got notified via GitHub about a new Onboarding Tracker (via cc: @camaraproject/admins) or do you need a separate mail to support@camaraproject.org about the issue?

Reviewer: @albertoramosmonagas Good point ... I proposed a new suggestion to split it even into three lines ... one for the request of wiki and mailing list and two if they are created. If ok for you we can proceed (I wasn't aware that this PR is still dangling around).

> - [ ] Create repository (by Admin, using automation)

Reviewer: One additional step needed - unavoidable as the repository is dependent on wiki and mailing list info and the wiki page points back to the (created) repository:

```suggestion
- [ ] Create repository (by Admin, using automation)
- [ ] Update API Repository wiki page with GitHub repository link
```

- [ ] Update README with API information (API owner)
- [ ] Update of CAMARA presentation and website (create issue within Marketing repository)
- [ ] Update of APIBacklog.md with new status ("Onboarded" or similar)
- [ ] Optional: get nominations of initial Maintainers (API owner)

> - [ ] Update Maintainers.md file with PR

Reviewer: Just seen that this step is also optional:

```suggestion
- [ ] Optional: Update Maintainers.md file with PR
```
